components/script/dom/csskeyframesrule.rs

```diff
@@ -101,6 +103,27 @@ impl CSSKeyframesRuleMethods for CSSKeyframesRule {
             self.rulelist().item(idx as u32)
         }).and_then(Root::downcast)
     }
+
+    // https://drafts.csswg.org/css-animations/#dom-csskeyframesrule-name
+    fn Name(&self) -> DOMString {
+        DOMString::from(&*self.keyframesrule.read().name)
+    }
+
+    // https://drafts.csswg.org/css-animations/#dom-csskeyframesrule-name
+    fn SetName(&self, value: DOMString) -> ErrorResult {
+        // https://github.com/w3c/csswg-drafts/issues/801
+        // Setting this property to a CSS-wide keyword or `none` will
+        // throw a Syntax Error.
+        match_ignore_ascii_case! { value,
+            "initial" => return Err(Error::Syntax),
+            "inherit" => return Err(Error::Syntax),
+            "unset" => return Err(Error::Syntax),
+            "none" => return Err(Error::Syntax),
+            _ => ()
+        }
```

Comment:
@emilio You're right. I fixed it to ascii case-insensitive. But I couldn't write it like  `"initial" || inherit` in match_ignore_ascii_case. Is there a way to reduce the code duplication?